src/stores/pacbioTubes.js

```diff
@@ -0,0 +1,50 @@
+import { defineStore } from 'pinia'
+import handlePromise from '@/api/PromiseHelper'
+import store from '@/store'
+
+export const usePacbioTubesStore = defineStore('pacbioTubes', {
+  state: () => {
+    return {
+      tractionTubes: [],
+    }
+  },
+  getters: {
```

Comment:
Do we need these at all? I'm just wondering if we could just use the store directly. I am not sure how many places these are used.